client/app/scripts/liveblog-edit/directives.js

```diff
@@ -214,6 +214,16 @@ define([
                 }
             };
         })
+        .directive('selectAllOnClick', [function() {
```

Comment:
IMHO select-all isn't a very good name for this, much better just selectOnCick or selectTextOnClick